cwms-data-api/src/main/java/cwms/cda/security/OpenIDAccessManager.java

```diff
@@ -87,7 +80,16 @@ private DataApiPrincipal getUserFromToken(Context ctx) throws CwmsAuthException
 
     private String getToken(Context ctx) {
         String header = ctx.header("Authorization");
-        return header.split("\\s+")[1];
+        if (header == null) {
```

Comment:
technically with the way the manager checking works this can't actually be null, though probably still worth the check just would likely be "illegal state" vs argument, at least for the header null check.